SimpactApp/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
I've noticed that both you and @anavianm created a barebones Django app - but in different folders (you used "SimpactApp" whike @anavianm used "simpact"). I suggest choosing and leaving one.

Reply:
Or did I get it wrong and you need both folders? Then maybe they can be renamed for clarity?